src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -134,13 +144,20 @@ final class OnboardingInfoViewController: UIViewController {
 		present(alert, animated: true, completion: nil)
 	}
 
-	private func updateUI() {
+	private func updateUI(exposureManagerState: ExposureManagerState) {
 		guard isViewLoaded else { return }
 		guard let onboardingInfo = onboardingInfo else { return }
 
 		titleLabel.text = onboardingInfo.title
 
-		imageView.image = UIImage(named: onboardingInfo.imageName)
+		let exposureNotificationsNotSet = exposureManagerState.status == .unknown && exposureManagerState.authorizationStatus == .unknown
```

Comment:
see my comment above ENManager.authorizationStatus ... you can have a look into ENStateHandler for details

Reply:
It's similar to your comment above. Please see my reply above.

Reply:
Yes please use it directly from the ENManager and not from wrapper class

Reply:
Based on my previous comment, this is outdated now. Please check [here](https://github.com/corona-warn-app/cwa-app-ios/pull/517/commits/e7ed188224d3aff072ed466b6c0b274e8290ee9c).